addons/cppcheckdata.py

```diff
@@ -139,6 +139,7 @@ class Token:
     Id = None
     str = None
     next = None
+    type = None
```

Comment:
I don't think type should be a public attribute. The addons should use isName, isOp, etc. I do not want to have these magic strings 'name', 'op', etc in the addons.